inttest/sonobuoy/terraform/gh.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -e
+sleep 30
+
+echo "prepare environment"
+sudo snap install jq
+sudo snap install kubectl --classic
+
+# Parse CLI args
+readonly git_tag="v0.7.0-beta1"
```

Comment:
Could we take this as input? And feed it via TF variables? This way it would be super easy to run conformance for any tag/version.